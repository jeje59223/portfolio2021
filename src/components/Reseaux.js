import React from "react";
import linkedin from "../images/linkedin.png";
import instagram from "../images/instagram.png";
import twitter from "../images/twitter.png";
import github from "../images/github.png";
import "../styles/reseaux.css";

const Reseaux = () => {
    return (
        <section id="reseaux">
            <div className="container my-5">
                <div className="row">
                    <div className="col-md-3 col-sm-6 text-center">
                        <a href="https://www.linkedin.com/in/jerome-cnockaert-359a20181/" target="_blank" rel="noreferrer">
                            <img src={linkedin} alt="linkedin" />
                        </a>
                    </div>
                    <div className="col-md-3 col-sm-6 text-center">
                        <a href="https://www.instagram.com/cnockaert59223/" target="_blank" rel="noreferrer">
                            <img src={instagram} alt="instagram" />
                        </a>
                    </div>
                    <div className="col-md-3 col-sm-6 text-center">
                        <a href="https://twitter.com/Jeje59223" target="_blank" rel="noreferrer">
                            <img src={twitter} alt="twitter" />
                        </a>
                    </div>
                    <div className="col-md-3 col-sm-6 text-center">
                        <a href="https://github.com/jeje59223" target="_blank" rel="noreferrer">
                            <img src={github} alt="github" />
                        </a>
                    </div>
                </div>
            </div>
        </section>
    )
}

export default Reseaux;