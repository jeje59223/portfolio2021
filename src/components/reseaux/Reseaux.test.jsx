import toJson from 'enzyme-to-json';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import Reseaux from './Reseaux';

configure({ adapter: new Adapter() });

describe('<Reseaux>', () => {
  it('should render Reseaux Component', () => {
    const wrapper = shallow(<Reseaux />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});
