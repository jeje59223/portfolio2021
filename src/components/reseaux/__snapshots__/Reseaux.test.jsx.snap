// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reseaux> should render Reseaux Component 1`] = `
<section
  id="reseaux"
>
  <div
    className="container my-5"
  >
    <div
      className="row"
    >
      <div
        className="col-md-3 col-sm-6 text-center"
      >
        <a
          href="https://www.linkedin.com/in/jerome-cnockaert-359a20181/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="linkedin"
            src="linkedin.png"
          />
        </a>
      </div>
      <div
        className="col-md-3 col-sm-6 text-center"
      >
        <a
          href="https://www.instagram.com/cnockaert59223/"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="instagram"
            src="instagram.png"
          />
        </a>
      </div>
      <div
        className="col-md-3 col-sm-6 text-center"
      >
        <a
          href="https://twitter.com/Jeje59223"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="twitter"
            src="twitter.png"
          />
        </a>
      </div>
      <div
        className="col-md-3 col-sm-6 text-center"
      >
        <a
          href="https://github.com/jeje59223"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="github"
            src="github.png"
          />
        </a>
      </div>
    </div>
  </div>
</section>
`;
